initializr-generator/src/test/groovy/io/spring/initializr/test/generator/PomAssert.groovy

```diff
@@ -97,7 +97,12 @@ class PomAssert {
 	}
 
 	PomAssert hasBootVersion(String bootVersion) {
-		assertEquals bootVersion, eng.evaluate(createRootNodeXPath('parent/pom:version'), doc)
+		// when using a custom parent, the bootVersion comes from a bom entry and not the parent pom version
+		if (!eng.evaluate(createRootNodeXPath('parent/pom:artifactId'), doc).equals("spring-boot-starter-parent")) {
```

Comment:
I don't think that's true. I guess we should rename that to `hasParentVersion` and leave the logic as it is. You could have an extra method `hasProperty("spring-boot.version", "1.3.5.RELEASE")` or something (if that doesn't exist yet)